static/partials/org-view.html

```diff
@@ -140,7 +140,9 @@ <h3>Organization Settings</h3>
 
               <time-machine-settings organization="organization"></time-machine-settings>
 
-              <quota-management-view quay-show="Features.QUOTA_MANAGEMENT" organization="organization" disabled="true"></quota-management-view>
+              <div quay-show="Features.QUOTA_MANAGEMENT">
```

Comment:
Is there a reason to switch to a div rather than the show parameter being inside the view object?